cylc/flow/workflow_files.py

```diff
@@ -1169,10 +1169,11 @@ def parse_reg(reg: str, src: bool = False, warn_depr=True) -> Tuple[str, Path]:
     """
     if not src:
         validate_workflow_name(reg)
+    cur_dir = reg.startswith(f'{os.curdir}{os.sep}')  # if starts with './'
```

Comment:
What is this short for?
I guess it's current directory, but if it is, that doesn't really match a data type of boolean IMO.
It's a bit of a mouthful, but I thing `reg_is_curdir` might be more helpful?